File: lua/refactoring/tests/refactor/106/csharp/prompt_func_param/extract.start.cs
namespace Test
{
    class Extract
    {
        public static int simpleFunction(int a)
        {
            int test = 1;
            int test_other = 1;

            for (int i = 0; i < test_other; i++) {
              Console.WriteLine("{0} {1}", a, test_other);
            }

            return test;
        }
    }
}

File: lua/refactoring/tests/refactor/106/lua/multiple-extract/extract.start.lua
-- stylua: ignore start

function simple_function(a)
    local test = 1
    local test_other = 11
    for idx = test - 1, test_other do
        print(idx, a)
    end

    for idx = test - 1, test_other do
        print(idx, test)
    end

    for idx = test - 1, test_other do
        print(idx, a)
    end
end
